XLForm/XL/Cell/XLFormBaseCell.h

```diff
@@ -35,7 +35,7 @@
 
 @property (nonatomic, weak) XLFormRowDescriptor * rowDescriptor;
 
--(XLFormViewController *)formViewController;
+@property (nonatomic, readonly, strong) XLFormViewController *formViewController;
```

Comment:
>@jkereako but it also allocates memory to the property state.

This isn't true.

The compiler automatically synthesizes instance variables (ivar) only if an accessor method is not implemented. Since an accessor method is implemented for `formViewController`, the compiler will not synthesize an ivar, hence, no memory is allocated.

Everything remains exactly the same except developers can now access `formViewController` using dot-syntax in addition the square brackets.

[Here's the documentation explaining ivar creation.](https://developer.apple.com/library/ios/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/EncapsulatingData/EncapsulatingData.html#//apple_ref/doc/uid/TP40011210-CH5-SW40)